examples/repudiable_authenticated_encryption.rs

```diff
@@ -15,19 +15,17 @@ use test_vectors::{ALICE_DID, ALICE_DID_DOC, ALICE_SECRETS, BOB_DID, BOB_DID_DOC
 
```

Comment:
I would suggest to keep `basic` name for the test. And provide same tests as python version

<img width="523" alt="image" src="https://user-images.githubusercontent.com/16944801/142252478-36262613-a3c9-40d8-b27b-332f47eba9d3.png">

Reply:
all tests except mediators provided